src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -516,6 +516,50 @@ Namespace Microsoft.CodeAnalysis.Operations
             End If
         End Function
 
+        Private Shared Function IsNestedDelegateCreation(operand As BoundNode, conversionKind As ConversionKind, conversionSyntax As SyntaxNode, targetType As TypeSymbol) As Boolean
+            If conversionKind = ConversionKind.Identity AndAlso operand.Kind = BoundKind.Parenthesized Then
+                Dim nestedOperand = operand
+                While nestedOperand IsNot Nothing And nestedOperand.Kind = BoundKind.Parenthesized
```

Comment:
>While nestedOperand IsNot Nothing And nestedOperand.Kind = BoundKind.Parenthesized [](start = 16, length = 82)

Consider using ```Do ... While``` statement. The enclosing ```If``` ensured that the condition is true for the first iteration. Also, the null check is unnecessary because BoundParenthesized.Expression can never be null, there is an assert in constructor.

Reply:
Done.

---
In reply to: [152430439](https://github.com/dotnet/roslyn/pull/23315#discussion_r152430439) [](ancestors = 152430439)